python/ray/autoscaler/_private/node_launcher.py

```diff
@@ -100,8 +101,9 @@ def _launch_node(self, config: Dict[str, Any], count: int, node_type: str):
 
         error_msg = None
         full_exception = None
+        created_nodes = {}
         try:
-            self.provider.create_node_with_resources(
+            created_nodes = self.provider.create_node_with_resources(
```

Comment:
This could also be None right?

Reply:
I think we should revert the changes since some node providers are not returning things (e.g. Kuberary, Fake Multinode)

Having the autoscaler depends on this return value is not gonna work for all providers.